backend/src/v4/models/group.js

```diff
@@ -323,7 +314,7 @@ Group.findByUID = async function (account, model, branch, revId, uid, showIfcGui
 	}
 
 	if (convertToIfcGuids) {
```

Comment:
Im confused as to why we have both showIfcGuids and convertToIfcGuids. Isnt getObjectsArrayAsExternalIds doing the same as getObjectIds with showIfcGuids set to true ?

Reply:
Not clear to me either, If i were to hazard a guess convertToIfcGuids makes an assumption that we have shared Ids and we're trying to convert to externals, and `showIfcGuids` implies we're turning it into Ifc guids no matter what.

We could change it, but in the interest of not breaking something i don't understand (and there's very little tests to check for regression), i'd rather not touch it.